music21/musicxml/xmlToM21.py

```diff
@@ -4194,9 +4241,12 @@ def xmlDirectionTypeToSpanners(
                     sp.startTick = mxObj.get('line-end')
                     sp.lineType = mxObj.get('line-type')  # redundant with setLineStyle()
 
-                start = spanner.SpannerAnchor()
-                self.insertCoreAndRef(totalOffset, staffKey, start)
-                sp.addSpannedElements(start)
+                self.spannerBundle.setPendingFirstSpannedElementAssignment(
+                    sp,
+                    'GeneralNote',
```

Comment:
we're going to get to this routine in a bit, but if it's possible to pass `note.GeneralNote` rather than the string `'GeneralNote'` it's the direction the library is going.  Sometimes it's not possible because of circular imports though.

Reply:
Ah, that's the way it always was, got it.